addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -6,17 +6,42 @@
     display: flex;
     flex-direction: row;
     justify-content: space-between;
-}
 
-.o_ChatterTopbar_buttonCount {
-    padding-left: 0.25rem !important;
+    .o_ChatterTopbar_buttonAttachmentsCountLoader {
+        margin-left: 2px;
+    }
+
+    .o_ChatterTopbar_buttonCount {
+        padding-left: 0.25rem !important;
+    }
+
+    .o_ChatterTopbar_buttonSendMessage {
+        padding: 0 16px;
+        border-left: none !important;
+    }
+
+    .o_ChatterTopbar_leftSection {
+        display: flex;
+    }
 }
 
 
+
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
+.o_ChatterTopbar {
+
+    .o_buttonActive {
+        color: #875A7B;
+        background-color: lighten(gray('300'), 7%);
+        border-color: #dee2e6;
+        border-bottom-color: transparent;
+        border-top-color: transparent;
```

Comment:
I think the appropriate way to do separation is not to hack borders on items, but have "separation items" with some style.

Reply:
I don't know how to deal with the intermittent bottom border so I went to the simplest solution for the moment